packages/Button/src/Button.tsx

```diff
@@ -3,11 +3,36 @@ import PropTypes from 'prop-types';
 import { colors } from '@leafygreen-ui/theme';
 import { emotion } from '@leafygreen-ui/lib';
 import { lighten, darken } from 'polished';
+import omit from 'lodash/omit';
 
 const { css, cx } = emotion;
 
-const buttonVariants = {
-  default: css`
+export enum Variant {
+  Default = 'default',
+  Primary = 'primary',
+  Info = 'info',
+  Danger = 'danger',
+  Dark = 'dark',
+}
+export enum Size {
```

Comment:
Small nit – Line break above here.

Eslint is supposed to auto-correct this sort of thing, but I don't think we have it properly configured for TS.